docs/versions.rst

```diff
@@ -5,6 +5,24 @@ Version History
 .. automodule:: more_itertools
    :noindex:
 
+10.6.0
+------
+
+* New functions:
+    * :func:`is_prime` and :func:`nth_prime` were added (thanks to JamesParrott and rhettinger)
+    * :func:`loops` was added (thanks to rhettinger)
+
+* Changes to existing functions:
+    * :func:`factor` and :func:`spy` were optimized (thanks to rhettinger)
```

Comment:
Perhaps make these entries more specific so that users can know how to take advantage of the improvements.

`factor` now handles much larger inputs and uses much less memory.

Call to `spy` now "self-flatten" so that they can be used in a loop without incurring additional overhead for each level of nesting.  Repeated spy lookaheads are now cheap.

 `polynomial_from_roots` is now non-recursive and can now handle much a larger number of roots.  Formerly, it would terminate with a `RecursionError`.